### ISSUE TITLE
Playbook attributes rendering ignores soft-set attributes

### ISSUE THREAD

**Issue by Grolribasi** [asciidoctor/asciidoctor-intellij-plugin#893]
Related to issue [#776](https://github.com/asciidoctor/asciidoctor-intellij-plugin/issues/776)

## Steps to reproduce

1. Define an attribute in the playbook like `:attribute: number 1@`.
2. Define an attribute on a page like `:attribute: number 2`.
3. Refer to an attribute on the page from step 2 like `{attribute}`
3. Attribute on a page is still rendered as "number 1".

## Expected behavior:

Attribute on step 3 is rendered as number 2.

## Environment

Plugin Version: 0.35.8

IntelliJ Details: 
IntelliJ IDEA 2021.2.3 (Community Edition)
Build #IC-212.5457.46, built on October 12, 2021
Runtime version: 11.0.12+7-b1504.40 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Windows 10 10.0
GC: G1 Young Generation, G1 Old Generation
Memory: 2044M
Cores: 8
Non-Bundled Plugins: com.github.jk1.ytplugin (2021.3.16), PlantUML integration (5.6.1), org.asciidoctor.intellij.asciidoc (0.35.8)
Kotlin: 212-1.5.31-release-546-IJ4638.7

**Comment by ahus1** [comment]
Pre-release 0.35.10 is now available that I tested with soft-setting attributes in component- and playbook-descriptors. The same logic applies for attributes set in plugin's settings. 

I needed to update this in several places, I kindly ask you to run tests if this and existing features work as expected.

I found that that `@` can be appended for the attribute's name and the attribute's value in the playbook and the component descriptor. The documentation describes only appending the `@` at the end of the attribute's values. 
The plugin supports both, still it will mark an `@` appended to the attribute name as a validation error in the YAML file.

The pre-release of the plugin is available from [GitHub releases](https://intellij-asciidoc-plugin.ahus1.de/docs/contributors-guide/user/installing-a-preview-release.html) and the [IntelliJ AsciiDoc EAP repository](https://intellij-asciidoc-plugin.ahus1.de/docs/contributors-guide/user/add-eap-repository-to-ide.html).
